src/main/java/recipeio/CommandValidator.java

```diff
@@ -65,8 +64,8 @@ public static boolean isParsableAsInteger(String input) {
     public static boolean isWord(String input) {
         // Regular expression to match only alphabetic characters
         if (!input.matches(CommandValidatorConstants.MATCH_WORD_REGEX)){
-            System.out.println("\tParameter cannot be parsed as an word.");
-            System.out.println("\tPlease enter a word using lower and upper case alphabets.");
+            System.out.println("Parameter cannot be parsed as an word.");
+            System.out.println("Please enter a word using lower and upper case alphabets.");
```

Comment:
If you have refactored the "Parameter cannot be parsed as an integer" string, please do so for the rest of the notifactions also. (word, date)